WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -231,7 +231,11 @@ def create
       registration_events_attributes: [:id, :event_id, :_destroy],
     ]
     if current_user.can_manage_competition?(competition_from_params)
-      permitted_params << :status
+      permitted_params << :accepted_at
+      status = params[:registration][:status]
+      if status
```

Comment:
Hmm, I've update the tests before adding this. So actually they shouls work with `status: :a` but that would require a change too. What do you prefer?